src/Html5ValidationField.tsx

```
@@ -137,11 +150,38 @@ class Html5ValidationField extends React.Component<Html5ValidationFieldProps> {
+      return React.createElement(component || 'input', {
+        ...fieldProps.input,
+        ref: mergedRef
+      })
+    }
+
     return React.createElement(Field, {
       ...fieldProps,
       validate: this.validate,
-      ref: innerRef as React.Ref<HTMLElement>,
-      component: 'input'
+      children: wrappedRender
```

Biome (2.3.13)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)